examples/invaders/simple-invaders/src/lib.rs

```diff
@@ -459,6 +459,43 @@ impl World {
             });
         }
     }
+
+    pub fn reset_game(&mut self) {
+        // Recreate the alien
+        self.invaders = Invaders {
+            grid: make_invader_grid(&self.assets),
+            stepper: Point::new(COLS - 1, 0),
+            direction: Direction::Right,
+            descend: false,
+            bounds: Bounds::default(),
+        };
+
+        // Empty laser
+        self.lasers.clear();
+
+        // Recreate the shield
+        self.shields = (0..4)
+            .map(|i| Shield {
+                sprite: Sprite::new(&self.assets, Frame::Shield1),
+                pos: Point::new(i * 45 + 32, 192),
```

Comment:
We should probably move these magic numbers to `const` values at global (module) scope. If they need to be reused in multiple modules, we can create a `crate::consts` module for them.

Reply:
LGTM